scripts/setup.js

```diff
@@ -52,7 +54,7 @@ const main = async () => {
     })
     console.log(explanation)
   }
-  let client = new faunadb.Client({ secret: adminKey })
+  let client = new faunadb.Client({ secret: adminKey, scheme: 'https', domain })
```

Comment:
Can you pull out the hardcoded 'scheme' please?

Reply:
👍